hugo/data/teams.yml

```diff
@@ -144,23 +152,29 @@ sections:
         github: https://github.com/gomesfernanda
 
 
-  - name: Developer Relations
+  - name: Marketing
     headline: Bringing the community and source{d} close together
     members:
       - picture: placeholder/male.png
-        name: Victor Coisne
-        position: Head of Developer Community
-        linkedin: https://www.linkedin.com/in/victorcoisne
+        name: Alex Dahlin
+        position: Marketing Intern
+        linkedin: https://www.linkedin.com/in/alexanderdahlin/
 
 
   - name: Operations
     headline: Taking care of the team at source{d}
+    members:
+      - picture: placeholder/male.png
+        name: Robbie Blue
```

Comment:
@eiso I saw in our organization chart that @bluer73 reports to you; Anna was in Operations team, so I added him here; should we move to a different team?